examples/plugins/job-pre-post/mail/avocado_job_mail.py

```diff
@@ -30,23 +96,145 @@ def __init__(self):
                                          key_type=str,
                                          default='localhost')
 
-    def mail(self, job):
-        # build proper subject based on job status
-        subject = '%s Job %s - Status: %s' % (self.subject,
-                                              job.unique_id,
-                                              job.status)
-        msg = MIMEText(subject)
-        msg['Subject'] = self.subject
+        if (hasattr(job.args, 'mail_job')):
+            state = getattr(job.args, 'mail_job')
+            if state == 'on':
+                self.enabled = 1
+            if state == 'off':
+                self.enabled = 0
+
+        if hasattr(job.args, 'mail_job_to'):
```

Comment:
You can simply use `getattr(job.args, "mail_job_to", None)`, the `getattr won't raise exception when default is supplied.